test/server.py

```diff
@@ -27,6 +28,12 @@
 from hyper.tls import NPN_PROTOCOL
 
 
+class SocketSecure(Enum):
+    SECURE = True
+    INSECURE = False
+    SECURE_NO_AUTO_WRAP = 'NO_AUTO_WRAP'
```

Comment:
Could you please elaborate on this?
I think we actually need the values to be able to convert boolean `secure` to an instance of this enum